backend/backend/services/business_logic.py

```diff
@@ -13,7 +13,7 @@ async def get_test_result(context: Context, subject: str):
     summary_chain = context.chains.get_chains()['summary_chain']
 
     knowledge = []
-    for iter in range(2):
+    for iter in range(1):
         queries = await adjacent_chain.ainvoke(subject, 4,
```

Comment:
Consider reducing the range in the for loop from 1 to 2 to improve code robustness.